training_docs/13_custom_datasets.md

```diff
@@ -0,0 +1,31 @@
+# Custom datasets
+
+Kedro supports a number of datasets out of the box, but you can also add support for any proprietary data format or filesystem in your pipeline.
+
+You can find further information about [how to add support for custom datasets](https://kedro.readthedocs.io/en/stable/07_extend_kedro/03_custom_datasets.html) in specific documentation covering advanced usage.
+
+## Contributing a custom dataset implementation
+
+One of the easiest ways to contribute back to Kedro is to share a custom dataset. Kedro has a :code:`kedro.extras.datasets` sub-package where you can add a new custom dataset implementation to share it with others. You can find out more in the [Kedro contribution guide](https://github.com/quantumblacklabs/kedro/blob/master/CONTRIBUTING.md) on Github.
```

Comment:
I'm not sure what `:code:` should do here?

Reply:
Me neither 😆 
This is another straight copy from the docs. I shall remove in both!